lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -583,6 +584,42 @@ void SessionStateEnforcer::m5g_process_response_from_upf(
   }
 }
 
+void SessionStateEnforcer::set_subscribed_qos(
+    SessionState& session_state, magma::SetSMSessionContextAccess* response) {
+  const auto& config = session_state.get_config();
+  auto* rsp = response->mutable_rat_specific_context()
+                  ->mutable_m5g_session_context_rsp();
+

```

Comment:
i think this is the same as the other pr 
Use a reference to save some lines
```
      config.rat_specific_context.m5gsm_session_context()
          .subscribed_qos()
```

Reply:
Modified the code as suggested.